src/App.jsx

```diff
@@ -1,65 +1,97 @@
-      {/* CATEGORIES COLUMN */}
-      <div className="CategoriesColumn col">
-        <div className="categories options">
-          <h2 className="title">Categories</h2>
-          {/* YOUR CODE HERE */}
-        </div>
-      </div>
+    const [activeCategory, setCategory] = React.useState("");
+    const [activeRestaurant, setRestaurant] = React.useState("");
+    const [activeMenuItem, setMenuItem] = React.useState("");
+    const NutritionContent = activeMenuItem != "" ? data.find((item) => item['item_name'] === activeMenuItem) : null;
+    return (
+        <main className="App">
+            {/*Categories sidebar goes here*/}
+            <div className="CategoriesColumn col">
+                <div className="categories options">
+                    <h2 className="title">Categories</h2>
+                    {categories.map((category, index) => (
+                        <Chip
+                            label={category}
+                            key={`category` + index}
+                            isActive={category === activeCategory}
+                            onClick={() => {
+                                setCategory(category);
+                            }}
+                        />
+                    ))}
+                </div>
+            </div>
 
-      {/* MAIN COLUMN */}
-      <div className="container">
-        {/* HEADER GOES HERE */}
+            {/* MAIN COLUMN */}
+            <div className="container">
+                {/* HEADER GOES HERE */}
+                <Header info={appInfo} />
 
-        {/* RESTAURANTS ROW */}
-        <div className="RestaurantsRow">
-          <h2 className="title">Restaurants</h2>
-          <div className="restaurants options">{/* YOUR CODE HERE */}</div>
-        </div>
+                {/* RESTAURANTS ROW */}
+                <div className="RestaurantsRow">
+                    <h2 className="title">Restaurants</h2>
+                    <div className="restaurants options">
+                        {restaurants.map((restaurant, index) => (
+                            <Chip label={restaurant} key={`category` + index} isActive={restaurant === activeRestaurant} onClick={() => (
+                              setRestaurant(restaurant)
+                            )}/>
+                        ))}
+                    </div>
+                </div>
 
-        {/* INSTRUCTIONS GO HERE */}
+                {<Instructions instructions={appInfo.instructions.start} />}
 
-        {/* MENU DISPLAY */}
-        <div className="MenuDisplay display">
-          <div className="MenuItemButtons menu-items">
-            <h2 className="title">Menu Items</h2>
-            {/* YOUR CODE HERE */}
-          </div>
+                {/* MENU DISPLAY */}
+                <div className="MenuDisplay display">
+                    <div className="MenuItemButtons menu-items">
+                        <h2 className="title">Menu Items</h2>
+                        {data
```

Comment:
Because there is a bit going on here, I would consider giving this it's own function. This will help with readability and future changes. Something like :

`  <h2 className="title">Menu Items</h2>
                        {renderMenuItems(data)}
                        
                        ...
                        // this function can live above in the component
                        const renderMenuItems = (menuItemList) => {
                        const filteredActiveItems = menuItemList.filter((item) => item['food_category'] === activeCategory && item['restaurant'] === activeRestaurant)
                        
                        return (<>{filteredActiveItems.map((item, index) => (<Chip label={item['item_name']} key={item['item_name']} 
                            isActive={item['item_name'] === activeMenuItem} onClick={
                              () => (
                                setMenuItem(item['item_name'])
                              )
                            }/>))}
                            </>);
                        }
                        
                        `